src/sklearn_evaluation/plot/feature_ranking.py

```diff
@@ -196,7 +198,11 @@ def feature_ranks_custom_algorithm(self, ranks):
         self._draw()
         return self.ax
 
+    def _set_default_colors(self, cmap):
```

Comment:
why is this needed? looks like it'll edit matplotlib's globals

Reply:
Good catch! removed it.